storage/byodb/external-buckets/index.md

```diff
@@ -91,9 +89,15 @@ WHERE `source` = "mql";
 ```
 
 {: .alert.alert-warning}
-**Note:** The dataset name is the name of the bucket you provided in the dialog (`users-reporting`), not that of the original dataset created in BigQuery 
+**Note:** The dataset name is the name of the bucket you provided in the dialog (`users-reporting`), not the original dataset created in BigQuery 
 (`sales_emea`). However, there are no technical limitations; they can have the same name. 
 
+## Sharing an External Dataset
+It is possible to share a Snowflake external bucket using the same process as [any other Storage bucket](https://help.keboola.com/catalog/#enable-sharing). Once the bucket is shared, the refresh operation is only available in the source project. Currently, it is possible to share entire buckets, not specific tables within them.
```

Comment:
`...source project (the project where the external dataset has been registered in)`

Reply:
bych trochu v te zavorce dovysvetlil co znamena "source project"